core/src/main/java/com/graphhopper/routing/util/DefaultEdgeFilter.java

```diff
@@ -40,6 +40,18 @@ public DefaultEdgeFilter(FlagEncoder encoder, boolean bwd, boolean fwd) {
         this.fwd = fwd;
     }
```

Comment:
Should we remove this or even all constructors? IMO the order bwd+fwd is unintuitive

Reply:
I removed the default constructor and made the other one protected. We could make it private, too and instead add a getter for the flag encoder (this is currently needed for the PrepEdgeFilter in PrepareRoutingSubnetworks).

Reply:
Changed the oder bwd+fwd too